amphp/artax/2017-05-09.yaml

```diff
@@ -0,0 +1,11 @@
+title:     Cookie leakage to wrong origins and non-restricted cookie acceptance
+link:      https://twitter.com/asyncphp/status/862033196824842246
```

Comment:
I would rather link to https://github.com/amphp/artax/releases/tag/v2.0.6 which gives some details about the vulnerability